sdk/management/docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,284 @@
+			// Two backend address pools which contain network interfaces for the virtual
+			//  machines to receive HTTP and HTTPS network traffic from the load balancer
+			// Two load balancing rules for HTTP and HTTPS to map public ports on the load
+			//  balancer to ports in the backend address pool
+			// Two probes which contain HTTP and HTTPS health probes used to check availability
+			//  of virtual machines in the backend address pool
+			// Three inbound NAT rules which contain rules that map a public port on the load
+			//  balancer to a port for a specific virtual machine in the backend address pool
+			//  - this provides direct VM connectivity for SSH to port 22 and TELNET to port 23
+
+			System.out.println("Creating a Internet facing load balancer with ...");
+			System.out.println("- A frontend IP address");
+			System.out.println("- Two backend address pools which contain network interfaces for the virtual\n" + "  machines to receive HTTP and HTTPS network traffic from the load balancer");
+			System.out.println("- Two load balancing rules for HTTP and HTTPS to map public ports on the load\n" + "  balancer to ports in the backend address pool");
+			System.out.println("- Two probes which contain HTTP and HTTPS health probes used to check availability\n" + "  of virtual machines in the backend address pool");
+			System.out.println("- Two inbound NAT rules which contain rules that map a public port on the load\n" + "  balancer to a port for a specific virtual machine in the backend address pool\n" + "  - this provides direct VM connectivity for SSH to port 22 and TELNET to port 23");
+
+			return Observable.merge(
+			Observable.just(indexable), azure.loadBalancers().define(loadBalancerName1).withRegion(region).withExistingResourceGroup(rgName)
+			// Add two rules that uses above backend and probe
+			.defineLoadBalancingRule(httpLoadBalancingRule).withProtocol(TransportProtocol.TCP).fromFrontend(frontendName).fromFrontendPort(80).toBackend(backendPoolName1).withProbe(httpProbe).attach().defineLoadBalancingRule(httpsLoadBalancingRule).withProtocol(TransportProtocol.TCP).fromFrontend(frontendName).fromFrontendPort(443).toBackend(backendPoolName2).withProbe(httpsProbe).attach()
+			// Add nat pools to enable direct VM connectivity for
+			//  SSH to port 22 and TELNET to port 23
+			.defineInboundNatPool(natPool50XXto22).withProtocol(TransportProtocol.TCP).fromFrontend(frontendName).fromFrontendPortRange(5000, 5099).toBackendPort(22).attach().defineInboundNatPool(natPool60XXto23).withProtocol(TransportProtocol.TCP).fromFrontend(frontendName).fromFrontendPortRange(6000, 6099).toBackendPort(23).attach()
+
+			// Explicitly define the frontend
+			.definePublicFrontend(frontendName).withExistingPublicIPAddress(publicIp).attach()
+
+			// Add two probes one per rule
+			.defineHttpProbe(httpProbe).withRequestPath("/").withPort(80).attach().defineHttpProbe(httpsProbe).withRequestPath("/").withPort(443).attach()
+
+			.createAsync());
```

Comment:
It is better to format this code. Same as below

Reply:
formatted using the editor

Reply:
the loadbalancer doesn't format, you could just copy the original code to here, I think